src/dom_components/model/ComponentVideo.js

```diff
@@ -182,8 +182,8 @@ export default Component.extend(
         {
           label: 'Poster',
           name: 'poster',
-          placeholder: 'eg. ./media/image.jpg',
-          changeProp: 1
+          placeholder: 'eg. ./media/image.jpg'
+          // changeProp: 1
```

Comment:
So the PR is just about changeProp in poster trait, right?

Reply:
yes

Reply:
> So the PR is just about changeProp in poster trait, right?

yes